module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,27 @@
+#include "compression.hpp"
+
+#include <algorithm>
+#include <utility>
+
```

Comment:
propozycja od @Kasperice, żeby kod był czytelniejszy:

using vectorOfPairs = std::vector<std::pair<uint8_t, uint8_t>>;
using arrayOfArrays = std::array<std::array<uint8_t, width>, height>;

Reply:
Szczerze powiedziawszy nie przepadam za takim skracaniem nazw typów, ale nie będę sę kłócić.

Reply:
więc jak wam zależy to piszcie, ale preferuje pełny typ

Reply:
ok, możemy zostawić pełny typ
może żeby kod był czytelniejszy to dać argumenty funkcji w osobnych linijkach?